baselines/diabetic_retinopathy_detection/jax_finetune_deterministic.py

```diff
@@ -128,11 +129,7 @@ def main(argv):
   # The pool is used to perform misc operations such as logging in async way.
   pool = multiprocessing.pool.ThreadPool()
 
-  def write_note(note):
-    if jax.process_index() == 0:
-      logging.info('NOTE: %s', note)
-
-  write_note('Initializing...')
+  vit_utils.write_note('Initializing...')
```

Comment:
Can we keep the write_note defined in the script? Internally we actually modify `write_note` to write additional logs, and the function's scoped to use an outside variable.  If we move the function to a separate module, we'd have to augment `write_note` to pass a second argument.

Reply:
Sure thing -- wasn't aware of the internal changes!